### ISSUE TITLE
Cant upload image which is latger than 1M(even less) 

### ISSUE THREAD

**Issue by upupzealot** [expressjs/multer#223]
When I choose to upload smaller images like 22k, it workds fine.
But when it comes larger, it dont work anymore, and left 2 temp files in the dst forlder on server, in random names.

my code is like this:
```js

var multer  = require('multer');
var upload = multer(
	{
		dest: './upload/images/',
		limits:{
    		fileSize: 10 * 1024 * 1024
    	}
    }
).single('file');
var fs = require('fs');

var checker = require('./check_util.js');

module.exports = function(app) {
	app.get('/upload', checker.checkLogin);
	app.get('/upload', function (req, res) {
		res.render(
			'upload',
			{
				user: req.session.user,
				success: req.flash('success').toString(),
				error: req.flash('error').toString()
			}
		);
	});

	app.post('/upload', checker.checkLogin);
	app.post('/upload', function (req, res) {
		upload(req, res, function (err) {
			if(err) {
				req.flash('error', err);
				res.redirect('/upload');
			}

			if(!req.file) {
				req.flash('error', 'upload flile cant be null');
				res.redirect('/upload');
			}

			if(req.file.size == 0) {
				fs.unlinkSync(req.file.path);
			} else {
				var target = './upload/images/' + req.file.originalname;
				fs.renameSync(req.file.path, target);
			}

			req.flash('success', target + ' uploaded!');
			res.redirect('/upload');
		});
	});
}

```

**Comment by oschn0r** [comment]
I had the same issue. I found out, that I used a fileFilter stub with the simple return of cb(null, true)

After I removed fileFilter everything works as expected. Uploads > 1M are working fine.

app.use(multer({ 
    dest: './temp/',
    limits: { fieldSize: 100},
    storage: storage
   //fileFilter: fileFilter
}).single("fileinput"));

function fileFilter (req, file, cb) {
	// To accept the file pass `true`, like so:
	cb(null, true)
}

**Comment by LinusU** [comment]
This should be fixed by #205 and released on npm as version `1.0.5` :rocket: 

Please reopen if issue persists.
